modules/SofaMiscTopology/SofaMiscTopology_test/CMakeLists.txt

```diff
@@ -2,18 +2,15 @@ cmake_minimum_required(VERSION 3.12)
 
 project(SofaMiscTopology_test)
 
-
+sofa_find_package(SofaBase REQUIRED)
 sofa_find_package(SofaMiscTopology REQUIRED)
 
 set(HEADER_FILES)
-set(SOURCE_FILES ../../empty.cpp)
+set(SOURCE_FILES TopologicalChangeProcessor_test.cpp)
 
-list(APPEND SOURCE_FILES
-TopologicalChangeProcessor_test.cpp
-)
 
 add_definitions("-DSOFAMISCTOPOLOGY_TEST_SCENES_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/scenes\"")
 add_executable(${PROJECT_NAME} ${SOURCE_FILES})
-target_link_libraries(${PROJECT_NAME} SofaGTestMain SofaTest SofaMiscTopology)
+target_link_libraries(${PROJECT_NAME} SofaGTestMain SofaMiscTopology)
```

Comment:
The libs you need in SofaBase are missing. SofaBaseTopology?

Reply:
if SofaMiscTopology is already linking with SofaBaseTopology, isn't it enough?

Reply:
@guparan ?